packages/moonstone/Popup/Popup.js

```diff
@@ -171,7 +171,7 @@ const PopupBase = kind({
 	styles: {
 		css: componentCss,
 		className: 'popup',
-		publicClassNames: ['popup', 'reserveClose']
+		publicClassNames: ['closeButton', 'popup', 'reserveClose']
```

Comment:
Let's document this either on `css` or inline with rationale for its exclusion.

Reply:
It's possible this should be excluded from the publicClassNames, and the padding rules should just be applied to `Popup`, rather than explicitly on `Dialog`, so they match. We don't have specific specs for `Popup` in this regard, so this probably makes sense.

What do you two think?

Reply:
It would be nice if `Popup` shares the same padding rules, but It's probably safer to not assume that `Popup` will be the same as `Dialog`.